cmake/Modules/FindLIBRT.cmake

```diff
@@ -1,17 +1 @@
-find_path(LIBRT_INCLUDE_DIRS
-  NAMES time.h
-  PATHS ${LIBRT_ROOT}/include/ ${KOKKOS_LIBRT_DIR}/include
-)
-
-find_library(LIBRT_LIBRARIES rt
- PATHS ${KOKKOS_HWLOC_DIR}/lib ${LIBRT_ROOT}/lib
-)
-include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(LIBRT DEFAULT_MSG LIBRT_LIBRARIES LIBRT_INCLUDE_DIRS)
-mark_as_advanced(LIBRT_INCLUDE_DIRS LIBRT_LIBRARIES)
-
-add_library(Kokkos::librt UNKNOWN IMPORTED)
-set_target_properties(Kokkos::librt PROPERTIES
-  IMPORTED_LOCATION "${LIBRT_LIBRARIES}"
-  INTERFACE_INCLUDE_DIRECTORIES "${LIBRT_INCLUDE_DIRS}")
-
+KOKKOS_FIND_IMPORTED(LIBRT HEADER time.h   LIBRARY rt)
```

Comment:
This refactoring probably makes me _almost_ as happy as it makes you.

Reply:
Just to get it functionally working. The logic routes through helper functions - so we can swap out exactly how this works later.

Reply:
And by just functionally working, I mean, yes, I should have just added a 
````
@KOKKOS_TPL_EXPORTS@
````
to the config file and appended to the string `KOKKOS_TPL_EXPORT` instead of a bunch of file operations.

Reply:
@DavidPoliakoff Actually, can you change your review to "changes requested"?

Reply:
@jjwilke Was this meant to target the other comment?

Reply:
In either case, I've yelled at you on your request